src/components/Transition/Transition.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react'
```

Comment:
This file has been removed some time ago in a refactor. Could you remove this file?
See the current components in the dev tree:
https://github.com/Provincie-Zuid-Holland/Omgevingsbeleid-Frontend/tree/dev/src/components